docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+     (shown on the right in Figure 1) is also used to implement the ASCS Windows
+    Server Failover Cluster (WSFC). The application server connection to the
+    highly available ASCS is through the cluster virtual network name. Assigning
+    the cluster virtual network name to the endpoint of this internal load
+    balancer is optional.
+
+-	**NICs.** Network interface cards (NICs) enable all communication of virtual
+    machines on a virtual network.
+
+-   **Network security groups**. To restrict traffic between the various subnets
+    in the virtual network, you can create [network security
+    groups][nsg] (NSGs).
+
+    
+-	**VPN.** To extend on-premises networks into Azure over a dedicated private
+    connection, a VPN is created. See our [planning and design][planning] recommendations.
+
+    > [!NOTE]
+    > [ExpressRoute][expressroute] is the recommended Azure service for creating private connections that do not go over the public Internet.
+    > 
+   
+-   **Jumpbox**. In the management subnet, this secure virtual machine on the
+    network (also called a bastion host) allows administrators to connect to the
+    other virtual machines for administrative tasks only.
+
+-   **Azure virtual machines**. Windows Azure virtual machines are used for
+    administration and SAP workloads. The database runs on Linux Azure
+    virtual machines.
+
+-   **Azure Storage**. This is a persistent store. The operating system, SAP
+    application, and business data are stored in [Azure Storage][azure-storage].
+
+-   **SIOS DataKeeper**. To support the WSFC environment, SIOS DataKeeper
+    Cluster Edition performs the cluster shared volume (CSV) function by
+    replicating independent disks owned by the cluster nodes. For details, see
+    “3. Important Update for SAP Customers Running ASCS on SIOS on Azure” at
+    [Running SAP applications on the Microsoft platform][running-sap].
+
+-   **SAP NetWeaver**. This application layer performs business logic
+    computation and includes SAP ASCS and the application servers.
+
+-   **Database tier**. SAP supports multiple database technologies. This document 
+    focuses on SAP HANA. The SAP HANA in-memory data platform provides high availability at
+    the data tier through the HANA System Replication (HSR) feature, which
+    implements a manual failover.
+ 
+
+## Recommendations
+Your requirements might differ from the architecture described here. Use these recommendations as a starting point.
+
+### SAP application on Azure
+The common practice for achieving high availability is through redundancy of
+critical components. In this distributed installation of the SAP application on
+a centralized database, the base installation is replicated to achieve high
+availability. For each layer of the architecture, the high availability design
+varies depending on the components as follows:
+
+-	**ASCS.** For high availability of ASCS on Azure Windows virtual machines,
```

Comment:
Should we link to [this doc](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/planning-guide#high-availability-for-the-sap-ascs-instance-on-windows) which describes the shared volume (and points to a relevant whitepaper)